org.opent2t.sample.hub.superpopular/org.opent2t.sample.hub.superpopular.raml

```diff
@@ -99,3 +99,37 @@ traits:
                 }
              }
     
+/subscribe:
+  description: |
+    This resource manages subscriptions to platoforms connected to this hub.
+  displayName: Subscribe
+    
+  post:
+    description: |
+       Verifies a subscription to a platform on this hub.
+    body:
+      application/json:
+        schema: Subscription
+        example: |
+          {
+            "callbackUrl": "http://myserver.com",
```

Comment:
callbackUrl is actually unneeded here, but verificationRequest is, so I'm adding it.

---
In reply to: [92022793](https://github.com/openT2T/translators/pull/136#discussion_r92022793) [](ancestors = 92022793)